tests/CMakeLists.txt

```diff
@@ -61,12 +61,6 @@ foreach(source_file ${source_files})
     endif()
 
     target_include_directories(${test_executable} PRIVATE
-        ${FYAML_INCLUDEDIR}
-        ${ARGP_INCLUDEDIR}
-        ${STATGRAB_INCLUDEDIR}
-        ${LZ4_INCLUDEDIR}
-        ${BZIP2_INCLUDEDIR}
-        ${ZLIB_INCLUDEDIR}
```

Comment:
I'm not so sure about ripping these out entirely.  It might be still needed if e.g. the user manually provided `FYAML_INCLUDEDIR` and so on.  But it seems, at least in many cases like `BZIP2_INCLUDEDIR` and `ZLIB_INCLUDEDIR` the pc files generated by homebrew (since they use `keg_only :provided_by_macos`) end up including system header include paths and this translates to things like `-I/Library/Developer/CommandLineTools/SDKs/MacOSX14.sdk/usr/include
` being passed to the compiler, which is bad and annoying.

So it really only makes sense if these contain custom user-provided paths, I think.  @jhunkeler any thoughts on this?